master/static/srv/entrypoint.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
 
+# When the task container is invoked via SLURM, we have
+# to set the slot IDs.
+if [ "$DET_RESOURCES_TYPE" == "slurm-job" ]; then
+    export DET_SLOT_IDS="[${CUDA_VISIBLE_DEVICES}]"
```

Comment:
maybe drop a comment explaining that slurm sets `CUDA_VISIBLE_DEVICES`

Reply:
also, we probably need this in every entrypoint, so maybe pull it into a script and use that?

Reply:
Addressing on dispatcher branch.